swift_browser_ui_frontend/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "swift_browser_ui_frontend_npm",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "start": "serve -l 8080 -s dist",
```

Comment:
are we planning to decouple `backend` from `frontend` or why do we have the `start` here ?

Reply:
The start script/command in the package.json file is something that's autogenerated by vue-cli, it can be, and has been removed in the latest push.